classroom.rb

```diff
@@ -0,0 +1,13 @@
+class Classroom
+  attr_accessor :label
+
```

Comment:
- [ ] nice work. please kindly add setter and getter for @student
eg
`attr_reader :students`

Reply:
Hi @tufoinnkuo10 thanks for the reminder 😄 .